netlify/pages/elements.yaml

```
@@ -0,0 +1,153 @@
+path: /open-source/elements
+title: Stoplight Elements
+relatedTags:
+  - blog-elements
+hero:
+  title: Open Source API Documentation Building Blocks
+  subtitle: >-
+    Build highly customized, interactive API Docs with embeddable web components generated from OpenAPI documents.
+  aligned: center
+  features:
+    - title: Open Source
+    - title: Maintained by Stoplight
+    - title: Free Forever
+  ctas:
+    - submit:
+        button:
+          title: Get Early Access
+          color: blue2
+        formId: 22bf0180-f3dd-4113-ac29-f68a7f408cb4
+        input:
+          placeholder: Your work email...
+          type: email
+      title: Get the API Design Guide
+      type: submit
+bodyImage:
+  alt: Elements
+  src: /images/elements/elements-new.png
+subFeatures:
+  title: Embed API Docs Anywhere
+  description: >-
+    Whether you have an existing developer portal or are working locally, you can use these components to add an
+    automatically generated API reference that fits seamlessly.
+  features:
+    - category:
+      title: Best-in-class OpenAPI Support
+      description: >-
+        Generate comprehensive API References from OpenAPI documents with support for advanced features like alloff,
+        ref, discriminators and much more.
+      image: /images/elements/openapi.svg
+    - category:
+      title: Interactive API Console
+      description: >-
+        Help developers get started faster by letting them test requests and responses within the documentation.
+      image: /images/elements/terminal.svg
+    - category:
+      title: Code Samples
+      description: >-
+        Do the heavy lifting for developers and kickstart integration by providing code samples in popular languages
+        like Curl, Python, Ruby, Java, and many more.
+      image: /images/elements/code.svg
+    - category:
+      title: Enhanced Markdown Support
+      description: >-
+        Write rich tutotials and descriptions with Stoplight-flavored markdown that gives the ability to add tabs, callouts, embed
```

```suggestion
        Write rich tutotials and descriptions with Stoplight-flavored Markdown that gives the ability to add tabs, callouts, embed
```

Usual advice is to try and use capitals for proper nouns, like Markdown, JSON Schema, etc. The sort of nerds who care about standards care about that sort of thing _a lot_. :)